client/client.go

```diff
@@ -1,19 +1,307 @@
-package client
+package main
 
 import (
 	"context"
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/json"
+	"encoding/pem"
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	log "github.com/golang/glog"
 
 	"github.com/openconfig/bootz/proto/bootz"
+	"go.mozilla.org/pkcs7"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/protobuf/proto"
 )
 
-type Client struct {
-	bootz.BootstrapClient
+// Represents a 128 bit nonce.
+const nonceLength = 16
+
+var (
+	secureOnly = flag.Bool("secure_only", true, "Whether to start the emulated device in SecureOnly boot mode.")
```

Comment:
nit:

so for defaults generally follow the "go default" so in this case a bool is default false
so your flag would be 
--insecure_boot

Reply:
Done.